modal/cli/_download.py

```diff
@@ -37,9 +37,12 @@ async def producer():
             if is_pipe:
                 await q.put((None, entry))
             else:
-                start_path = os.path.dirname(remote_path).split("*")[0]
-                rel_path = Path(entry.path).relative_to(start_path.lstrip("/"))
-                output_path = local_destination / rel_path
+                start_path = str(PurePosixPath(remote_path).parent).split("*")[0]
```

Comment:
Think this should work too

```suggestion
                start_path = Path(remote_path).parent.as_posix().split("*")[0]
```

Reply:
Yep I'll add tests in a bit